components/home/sections/contact.js

```diff
@@ -1,98 +1,118 @@
+"use client";
 import { PYCONUG_EMAIL } from "@/utils/constants";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
```

Comment:
not really necessary to keep calling `import "react-toastify/dist/ReactToastify.css";` on every usecase....can be configured to apply styles globally...

```suggestion
 ```

Reply:
Okay, so let me wrapp it in the layout of the app which i think resolves this message